scripts-python/bless_test_results

```diff
@@ -135,11 +138,13 @@ def bless_history(test_name, baseline_tag, baseline_dir_for_test, testcase_dir_f
     acme_root = run_cmd("./xmlquery CESMSCRATCHROOT -value", from_dir=testcase_dir_for_test)
 
     case = os.path.basename(testcase_dir_for_test)
-    cime_root = cime_util.get_cime_root()
+    cime_root = CIME.utils.get_cime_root()
     compgen = os.path.join(cime_root, "scripts", "Tools", "component_compgen_baseline.sh")
     machine_env = os.path.join(testcase_dir_for_test, ".env_mach_specific.sh")
     run_dir = os.path.join(acme_root, case, "run")
-    cprnc_loc = cime_util.get_machine_info("CPRNC_ROOT")
+    machobj = Machines()
+    machobj.probe_machine_name()
```

Comment:
How about having Machine objects automatically probe for machine if user does not give them one?